reaktive/src/commonTest/kotlin/com/badoo/reaktive/observable/SkipTest.kt

```diff
@@ -0,0 +1,41 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.testutils.TestObservable
+import com.badoo.reaktive.testutils.test
+import com.badoo.reaktive.testutils.values
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class SkipTest {
+    @Test
+    fun should_skip_n_values() {
+        val source = TestObservable<Int>()
+        val observer = source
+            .skip(2)
+            .test()
+
+        source.onNext(1)
+        source.onNext(2)
+        source.onNext(3)
+        source.onNext(4)
+
+        assertEquals(2, observer.events.size)
```

Comment:
Hi.
I have an honest doubt.

This assert should not be comparing the size of events list with 3?

I am asking this because I looked at the code and saw that the event list contains 3 kinds of events, OnNext, OnComplete, and OnError.

So, if the observer emitted only 2 items (3 and 4), and then completed, I believe that the size of the list should be 3. Something like `[OnNext(3), OnNext(4), OnCompleted]`

That makes sense?